### ISSUE TITLE
CE-273 Ensure heading levels increase by one.

### ISSUE THREAD

**Issue by Paul-Clue** [LLK/scratch-www#7177]
### Where To Find Bug
`/ideas`, `/about`, `/messages`, `/mystuff`, `/accounts`

### Expected Behaviour
The WCAG standard for decreasing heading levels (headers should decrease by one)  help screen reader users better understand a page's content. For example, every page should have only one h1 level heading that signifies what the content on the entire page is about. h2 headers should come after and represent the main sections of the main idea of the page.

### Actual Behaviour
While tabbing through the page, the screen reader user will notice that some heading levels are not in a proper order, leading to possible disorientation and confusion about the content.

### Steps To Reproduce

- Open https://scratch.mit.edu/ in any browser on your computer
- Turn-on your screen reader of choice
- Start tabbing through the page.
- Notice that the heading levels are not in the correct order

### System Details
macOs Ventura, Chrome Browser, VoiceOver (AT)
